src/plugins/intel_gna/transformations/remove_converts.cpp

```diff
@@ -72,8 +73,11 @@ namespace GNAPluginNS {
                 return false;
             }
 
-            // the result presicion will be changed automaically
-            ngraph::replace_output_update_name(convert_node->output(0), convert_node->input_value(0));
+            // the result precision will be changed automatically
+            ngraph::replace_node(convert_node, output_node);
+            output_node->set_friendly_name(convert_node->get_friendly_name());
+            ngraph::copy_runtime_info(convert_node, output_node);
+
```

Comment:
actually `replace_output_update_name` should preserve output names for old api which is based on node names. But looks like in some cases it doesn't work as expected and prevent potential eliminations. In general in `replace_output_update_name` we also have to check that `replacement` consumers are not results but not just `replacement.get_node()->get_users().size() == 1` also we need to consider number of output ports.
So, could you fill a bug for it?

Reply:
Sure, 76727

Reply:
reverted wa